src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -118,18 +118,22 @@ void TxDetailDialog::setData(WalletModel *model, const QModelIndex &index){
 
 }
 
-void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx){
+void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx) {
     this->model = model;
     this->tx = &tx;
     CAmount txFee = tx.getTransactionFee();
     CAmount totalAmount = tx.getTotalTransactionAmount() + txFee;
 
     ui->textAmount->setText(BitcoinUnits::formatWithUnit(nDisplayUnit, totalAmount, false, BitcoinUnits::separatorAlways) + " (Fee included)");
-    if(tx.getRecipients().size() == 1){
+    if (tx.getRecipients().size() == 1) {
         ui->textSend->setText(tx.getRecipients().at(0).address);
+        ui->textSendLabel->setText(tx.getRecipients().at(0).label);
+        ui->labelOutputs->setVisible(true);
         ui->pushOutputs->setVisible(false);
-    }else{
+    }
+    else {
```

Comment:
styling nit: I think the (most widely used) convention in the codebase is to put the else on the same line of the closing bracket (with spaces) `} else {`

Reply:
Automatic formatting of IDE adjusted